.github/workflows/ci.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: codespell
         if: matrix.run_doc == true
         run: |
-          uv run codespell
+          uv run codespell --skip "*.lock"
```

Comment:
This should be in pyproject.toml, otherwise check_ci.sh will not work identical to CI, which is kind of it´s purpose.